ide/static/ide/js/autocomplete.js

```diff
@@ -126,72 +134,135 @@ CloudPebble.Editor.Autocomplete = new (function() {
     var mLastAutocompleteToken = null;
     var mLastAutocompletePos = null;
     var run_last = _.debounce(function() {
-        if(mLastInvocation) {
+        if (mLastInvocation) {
             console.log("running trailing completion.");
             self.complete.apply(self, mLastInvocation);
         }
     }, 50);
+
+    function collect_header_files() {
+        var local = _.map(_.keys(CloudPebble.Editor.GetAllFiles()).sort(), function(name) {
+            return {
+                name: name,
+                local: true
+            }
+        });
+        var lib = _.map(CloudPebble.Dependencies.GetHeaderFileNames().sort(), function(name) {
+            return {
+                name: name,
+                local: false
+            }
+        });
+        return _.filter(local.concat(lib), function(item) {
+            return item.name.indexOf('.h', item.length - 2) !== -1;
+        }).concat({name: 'pebble.h', local: false});
+    }
+
+    function search_header_files(command, editor) {
```

Comment:
I like `try_complete_header_files` or maybe `try_complete_includes` since it doesn't really deal with other kinds of macro. Which actually brings my attention to a bug: completions for macros actually did work before (although it just suggested possible macros). This gets rid of that, so I'll have to put it back in.